tests/ui/testem.js

```diff
@@ -0,0 +1,42 @@
+/*
+For copyright information, see the AUTHORS.md file in the docs directory of this distribution and at
+https://github.com/fluid-project/sjrk-story-telling/blob/master/docs/AUTHORS.md
+
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENSE.txt
+*/
+
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+fluid.setLogging(true);
+
+require("fluid-testem");
+
+fluid.defaults("fluid.tests.testem", {
+    gradeNames: ["fluid.testem.instrumentation"],
+    coverageDir: "coverage",
+    reportsDir: "reports",
+    testPages: ["tests/ui/all-tests.html"],
+    instrumentationOptions: {
+        nonSources: [
+            "./**/*.!(js)",
+            "./Gruntfile.js"
+        ]
+    },
+    sourceDirs: {
+        src: "src"
+    },
+    contentDirs: {
+        tests: "tests"
+    },
+    testemOptions: {
+        launch: "Headless Chrome,Headless Firefox",
```

Comment:
Browser tests run in Chrome and Firefox but not Safari and IE. Combining with the removal of `http://localhost:8081/tests/all-tests.html` request handler, is there a suggested way to run automated browser tests in non-chrome-and-firefox browsers?

Reply:
@cindyli testem doesn't have a runner for MS Edge (but it's basically Chrome now anyways), I don't believe we support IE, and there is an [issue](https://github.com/testem/testem/issues/1387) with the latest versions of Safari working with testem. 

Currently the way to run automated tests outside of testem is that you have to provide your own web server to serve the project and then navigate to tests/ui/all-tests.html. This is mentioned in the [README.md](https://github.com/jobara/sjrk-story-telling/blob/SJRK-122/README.md#browser-tests).

Reply:
Running `tests/ui/all-tests.html` with my web server hangs from the first test with the error `jQuery is not defined`.